src/components/ModelingMachineProvider.tsx

```diff
@@ -1043,38 +1052,85 @@ export const ModelingMachineProvider = ({
             }
           }
         ),
-        'Get convert to variable info': fromPromise(
+        'Apply named value constraint': fromPromise(
```

Comment:
I've had beef with calling them "variables" in our codebase because they don't change, but lmk if I should use that naming and I can make it `Apply named variable constraint` and the like everywhere

Reply:
and all variables are named by their very being.

I think the churn isn't worth the bikeshed